tabpy-server/setup.py

```diff
@@ -28,13 +28,27 @@
     package_data={'tabpy_server.static': ['*.*'],
                   'tabpy_server': ['startup.*', 'state.ini']},
     license='MIT',
+# Note: many of these required packages are included in base python
```

Comment:
Comments should be aligned with code according to pep8

Reply:
fixed